climada/hazard/storm_europe.py

```diff
@@ -830,3 +831,106 @@ def _hist2prob(self, intensity1d, sel_cen, spatial_shift, ssi_args=None,
         ssi = self.calc_ssi(intensity=intensity_out, **ssi_args)
 
         return intensity_out[:, sel_cen], ssi
+
+
+def generate_WS_forecast_hazard(run_datetime = dt.datetime.today().replace(hour=0, 
+                                                                        minute=0, 
+                                                                        second=0, 
+                                                                        microsecond=0),
+                             event_date = (dt.datetime.today().replace(hour=0, 
+                                                                       minute=0, 
+                                                                       second=0,
+                                                                       microsecond=0)
+                                           + dt.timedelta(days=2)),
+                             haz_model = 'icon-eu-eps',
+                             haz_raw_storage = None):
+    """ use parameters to generate needed forecast hazard either by download
+    or through file.
```

Comment:
Please make a clear and precise description.

Reply:
Done